plenum/server/propagator.py

```diff
@@ -285,15 +285,14 @@ def request_propagates(self, req_keys):
         :return:
         """
         i = 0
-        for (idr, req_id) in req_keys:
-            if (idr, req_id) not in self.requested_propagates_for:
-                self.request_msg(PROPAGATE, {f.IDENTIFIER.nm: idr,
-                                             f.REQ_ID.nm: req_id})
-                self._add_to_recently_requested((idr, req_id))
+        for digest in req_keys:
+            if digest not in self.requested_propagates_for:
+                self.request_msg(PROPAGATE, {f.DIGEST.nm: digest})
```

Comment:
Do we need only `digest` here? I think the code in MessageHandler expects all the fields...

Reply:
MessageHandler for Message Request with propagate was change. Now it work only with digest.